nextcloud_update.sh

```diff
@@ -309,7 +309,17 @@ fi
 # Update all Nextcloud apps
 if [ "${CURRENTVERSION%%.*}" -ge "15" ]
 then
-    occ_command app:update --all
+    UPDATED_APPS=$(occ_command app:update --all | sed '1~2d' | awk '{print $1}' | tr '\n' " ")
+    UPDATED_APPS=${UPDATED_APPS// /", "}
+fi
+
+# Check which apps got updated
+if [ -n "$UPDATED_APPS" ]
+then
+    UPDATED_APPS=${UPDATED_APPS:0:-2}
```

Comment:
this removes the latest two characters, in the string, which is needed because else you had a comma at the end which wouldn't be right. It needs to be here, because if the string is empty, then you get an error.